bower.json

```diff
@@ -2,7 +2,7 @@
   "name": "locate",
   "version": "2.4.10",
   "dependencies": {
-    "agrc": "agrc-widgets#~6.0",
+    "agrc": "agrc-widgets#6.1.6",
```

Comment:
should this be agrc.widgets?

Reply:
I think so since it's an old dojo project. Or am I missing something?

Reply:
Oh, now I see the build error. Strange, since it installed on my local machine without issues...